src/components/Person.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import './Person.scss';
+
+import PropTypes from 'prop-types';
+
+export const Person = ({
+  name,
+  age,
+  sex,
+  isMarried,
+  partnerName,
+}) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${name}`}
+    </h2>
+    {age ? (
+      <p className="Person__age">
+        { `I am ${age}`}
+      </p>
+    ) : null}
```

Comment:
```suggestion
    {age && (
      <p className="Person__age">
        { `I am ${age}`}
      </p>
    )}
```
Conditional rendering можна використовувати ось так, якщо є тільки один варіант - рендерити щось чи ні